app/_src/policies/meshloadbalancingstrategy.md

```diff
@@ -448,9 +448,9 @@ spec:
         localityAwareness:
           localZone:
             affinityTags:
-              - key: infra.io/datacenter
+              - key: kubernetes.io/hostname
```

Comment:
this is still missing in this PR https://github.com/kumahq/kuma/pull/9690/files - @lukidzi - any reason for not including it?

Reply:
Based on https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/#node-labels I noticed only zone and region

Reply:
PR added https://github.com/kumahq/kuma/pull/10243

Reply:
We need to change all examples. I think there is more examples with tags

Reply:
there is only this https://github.com/kumahq/kuma-website/blob/c6b449fdd7331518893e8449a4c30fde52e9108c/app/_src/policies/meshloadbalancingstrategy.md#L348-L349